src/main/java/com/derongan/minecraft/deeperworld/DeeperWorld.kt

```diff
@@ -34,5 +37,17 @@ class DeeperWorld : JavaPlugin() {
 
         //register command executor
         DeeperCommandExecutor
+
+        if(DeeperConfig.data.maxSafeFallingDistance != -1f){
+            schedule{
+                repeating(20)
+                while(true){
+                    Bukkit.getServer().onlinePlayers.forEach{
```

Comment:
You have access to server within the Plugin's context so could do
```suggestion
                    server.onlinePlayers.forEach{
```